tutorials/milvus_lite_arize.ipynb

```diff
@@ -0,0 +1,1428 @@
+{
```

Comment:
<p>How complex is the code needed to wrangle this JSON file from the original assets? If it's relatively straightforward, let's see if we can include it here rather than persisting an additional dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>It's a very straight forward task but hard to comprehend as i instantiated the storage context as built in the llama_index_search_and_retrieval notebook, persisted it locally and from various json sources that were within those I built this json file that has milvus vector structure from the fields obtained</p>

Reply:
<p>at the very least currently I eliminated last for loop</p>